src/index.js

```diff
@@ -27,21 +88,25 @@ class Dropzone extends React.Component {
   componentDidMount() {
     this.enterCounter = 0;
     // Tried implementing addEventListener, but didn't work out
-    document.body.onfocus = this.onFileDialogCancel;
+    if (document.body instanceof HTMLElement) {
+      document.body.onfocus = this.onFileDialogCancel;
+    }
   }
 
   componentWillUnmount() {
     // Can be replaced with removeEventListener, if addEventListener works
-    document.body.onfocus = null;
+    if (document.body instanceof HTMLElement) {
+      document.body.onfocus = null;
+    }
   }
 
-  onDragStart(e) {
+  onDragStart(e: Object) {
```

Comment:
We can use `SyntheticDragEvent` to be more specific.

[https://github.com/facebook/flow/blob/v0.35.0/lib/react.js#L349](https://github.com/facebook/flow/blob/v0.35.0/lib/react.js#L349)